django_cloud_deploy/cli/new.py

```diff
@@ -119,7 +125,8 @@ def main(args: argparse.Namespace, console: io.IO = io.ConsoleIO()):
         'project_creation_mode': workflow.ProjectCreationMode.CREATE,
         'bucket_name': getattr(args, 'bucket_name', None),
         'service_accounts': getattr(args, 'service_accounts', None),
-        'services': getattr(args, 'services', None)
+        'services': getattr(args, 'services', None),
```

Comment:
where does services come from?

Reply:
I did not modify 'services'. I added 'appengine_service_name' and this can be provided by a SimpleNamespace object in e2e test. See https://github.com/GoogleCloudPlatform/django-cloud-deploy/blob/master/django_cloud_deploy/tests/e2e/gke_deploy_test.py#L107.